site/src/examples/pill/Disabled.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from "react";
+import { PillNext } from "@salt-ds/lab";
+
+const handleClick = () => {
+  console.log("clicked");
+};
+
+export const Disabled = (): ReactElement => (
+  <PillNext disabled onClick={handleClick}>
```

Comment:
When the PIll is disabled, the `onClick` handler will never be fired, so this code is redundant. It might therefore make the example clearer if it only used the `disabled` prop and omits the rest.

```suggestion
export const Disabled = (): ReactElement => (
  <PillNext disabled>
```